frontend/helpers/tillDuration.ts

```diff
@@ -25,3 +26,20 @@ export const formatInterval = function (startDate: Date, endDate: Date): string
         });
     }
 };
+
+export const calculateTimeTillProfitable = function (auction: AuctionTransaction) {
```

Comment:
- [ ] I suggest to rename this function to `calculateTransactionProfitDate` which returns:
  - `undefined` in case it's not possible to compute
  - `Date` in the future when it will become profitable
  - `Date` in the past if it is already profitable

- [ ] Please move this function to the `core/price.ts` 
- [ ] Please add `transactionProfitDate` to each auction
- [ ] On the frontend, you only need to use this date and don't need to compute anything